whylog/log_reader/__init__.py

```diff
@@ -91,3 +110,11 @@ def investigate(self, original_front_input):
                     "Cannot operate on %s which is different than %s" % (host, "localhost")
                 )
         return clues
+
+    @classmethod
+    def _generate_tmp_files(cls, tmp_assigned):
```

Comment:
this method needs to be called from inside `log_type.files_to_parse(forced_logtype=None)`